app/Pdf/YetiForcePDF.php

```diff
@@ -328,6 +335,7 @@ public function parseVariables(string $str)
 		$textParser = \App\TextParser::getInstanceById($this->recordId, $this->moduleName);
 		$textParser->setType('pdf');
 		$textParser->setParams(['pdf' => $this->moduleModel]);
+		$textParser->setInventoryColumns($this->getInventoryColumns());
```

Comment:
$textParser->setParams(['pdf' => $this->moduleModel], 'inventoryColumns' =>$this->getInventoryColumns());